markdown/nextflow_tutorial.md

```diff
@@ -39,50 +39,45 @@ A process will be initiated once all the elements of the channels defined in the
 ## Scripts
 Each nextflow process ***must end*** with a **script** block, which defines the code to be executed by the process.
 It can call on the variables from the input section and will release the output to the output section.
-A script is initiated either by:
+A script is initiated as follows:
 
 ```
 script:
-<user script>
-```
-
-or by triple quotations:
-
-```
 """
 <user script>
 """
 ```
 
-The triple quotations are required if one wants to use several lines of code without commenting out the line breaks, if only one line is to be executed it is possible to use single quotes.
-Importantly, strings delimited by a `"` character support variable substitutions, while strings delimited by `'` do not.
-Also, by using this method of initiating the script, it is possible to use all kinds of programming languages for execution.
-When the script is initiated using `script:` it will be assumed to be written in [**groovy**](http://groovy-lang.org), the underlying language of nextflow.
-The `script:` block can be used in addition to `"""<user script>"""`, which is mainly used if the format of the input needs to be changed before a script is run.
-In the `"""<user script>"""` block, one can define the scripting language by a leading shebang statement (i.e. !#/usr/bin/perl).
-If no shebang is specified, the command is performed as bash.
+The `script:` header can be omitted if using triple quote blocks, however we recommend always using it for clarity.
```

Comment:
I rewrote this a little, as it's a personal OCD annoyance of me when I see the script blocks without a `script:` definition. So I'd prefer to teach people to always use it 🤓